configure.ac

```diff
@@ -68,7 +68,7 @@ dnl ======================
 
 INPUT_PLUGINS="metronom psf tonegen vtx xsf"
 OUTPUT_PLUGINS=""
-EFFECT_PLUGINS="bitcrusher compressor crossfade crystalizer mixer silence-removal stereo_plugin voice_removal echo_plugin"
+EFFECT_PLUGINS="bitcrusher compressor crossfade crystalizer mixer silence-removal stereo_plugin voice_removal echo_plugin background_music"
```

Comment:
Keep this list sorted alphabetically, please also move `echo_plugin` while fixing that.

Reply:
Fixed as part of [this commit](https://github.com/audacious-media-player/audacious-plugins/commit/ff69402b28597d41ac73c29e04ec9dbfd79d5a8f)